packages/view3d/src/core/Renderer.ts

```diff
@@ -221,7 +223,12 @@ class Renderer {
     camera.updatePosition();
 
     scene.shadowPlane.render();
-    threeRenderer.render(scene.root, camera.threeCamera);
+
+    if (postProcessing.isPostProcessing) {
+      this._view3D.postProcessing.composer.render();
+    }else {
```

Comment:
missing space

Reply:
I'm waiting for https://github.com/naver/egjs-view3d/pull/48